packages/app/src/app/pages/common/Modals/StorageManagementModal/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import filesize from 'filesize';
+import { useOvermind } from 'app/overmind';
+import {
+  Container,
+  Title,
+  JustifiedArea,
+  SubTitle,
+  Description,
+  SubDescription,
+  Rule,
+  LoadingAnimationContainer,
+} from './elements';
+import FilesList from './FilesList';
+
+const StorageManagementModal = () => {
```

Comment:
```suggestion
export const StorageManagementModal: React.FC = () => {
```

We'd like to move everything to named exports instead, so let's change this line.